doc/examples/anim14/anim14.sh

```diff
@@ -79,12 +79,12 @@ gmt begin
 gmt end
 EOF
 # 2. Set up main script
-cat << EOF > main.sh
+cat << 'EOF' > main.sh
```

Comment:
The main script in anim14 uses both movie-specific variables (e.g., `${MOVIE_COL0}`) and general bash variables (e.g., `$W`). 

I think you have to create a `inc.sh` file to make it work.